jsx/Modal.js

```diff
@@ -62,7 +62,7 @@ class Modal extends Component {
    * @return {JSX} - React markup for the component
    */
   render() {
-    const {show, children, onSubmit, title} = this.props;
+    const {show, children, onSubmit, title, width} = this.props;
```

Comment:
add width to proptype definition

Reply:
not sure when this was resolved but I just went to fix it and it's already there